docker-compose.yml

```diff
@@ -14,7 +14,9 @@ services:
       - ./postgres/vector_extension.sql:/docker-entrypoint-initdb.d/0-vector_extension.sql
     env_file:
       - .env
-
+    networks:
+      inner:
+        ipv4_address: 192.168.0.4
```

Comment:
DO FUCKING NOT
use built-in DNS (if you're using podman, you'll require one addon to make it work)